app/ingestion/jd_mapping.json

```diff
@@ -0,0 +1,43 @@
+{
+  "mappings": {
+    "job_description": {
+      "properties": {
+        "companyName": {
+          "type": "keyword",
+          "store": false,
```

Comment:
by default, fields are not stored. So, you dont have to explicitly map this too..